docs/src/use/configure/index.md

```diff
@@ -49,7 +49,10 @@ All of these options give you fine-grained control over how ESLint treats your c
 
 * [Configure Plugins](./plugins#configure-plugins)
 * [Specify a Processor](./plugins#specify-a-processor)
-* [Configure Parsers](./plugins#configure-a-parser)
+
+[**Configure a Parser**](./parser)
+
+* [**Configure a Custom Parser**](./parser#configure-a-custom-parser)
```

Comment:
This is the only sub-item that's bolded. Was that intentional? If not:

```suggestion
* [Configure a Custom Parser](./parser#configure-a-custom-parser)
```

Reply:
not intentional, thx for the catch!